runtime/common/xwalk_content_client.cc

```diff
@@ -1,3 +1,4 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
```

Comment:
What do you mean? It sounded like if the code comes from upstream the copyright notice should not be preserved, which I guess is not what you intended to convey.